modules/library/src/main/scala/zio/elasticsearch/aggregation/Aggregations.scala

```diff
@@ -125,6 +167,9 @@ private[elasticsearch] final case class Terms(
   def withSubAgg(aggregation: SingleElasticAggregation): TermsAggregation =
     self.copy(subAggregations = aggregation +: subAggregations)
 
+  private[elasticsearch] def paramsToJson: Json =
+    Obj(name -> paramsToJsonHelper)
```

Comment:
This is a bit strange as it looks like a "helper" is what is being included in the json. Would the following renaming make sense?
- paramsToJson -> toJson
- paramsToJsonHelper -> jsonParams